qiskit/transpiler/passes/scheduling/alignments/align_measures.py

```diff
@@ -86,6 +87,15 @@ class AlignMeasures(TransformationPass):
         However, it may return meaningless measurement data mainly due to the phase error.
     """
 
+    @deprecate_func(
+        # TODO: This grammar doesn't make sense
```

Comment:
Ditto

Reply:
the grammar on this one makes sense to me, just a little imprecise?  It's saying that `ConstrainedReschedule` does the same thing, except that also satisfies some more constraints as well.  (I wouldn't swear to it, but I think this pass only considers alignment of measurements, whereas `ConstrainedReschedule` does both that and pulse-gate alignments simultaneously.)